_includes/author.html

```diff
@@ -12,7 +12,7 @@
                 {{ author.name }}
             {% if page.className != 'authors' %}</a>{% endif %}
         </h1>
-        <h4>{{ author.role }} · {{ author.tribe }} Tribe</h4>
+        {% if author.tribe and author.role %}<h4>{{ author.role }} · {{ author.tribe }} Tribe</h4>{% endif %}
```

Comment:
We still want roles for everyone right?

So:
<h4>{{ author.role }}%{ if author.tribe %} · {{ author.tribe }} Tribe{% endif %}</h4>
?